share/completions/code.fish

```diff
@@ -4,26 +4,24 @@ function __fish_complete_vscode_extensions
     command --quiet code; and code --list-extensions
 end
 
-complete -c code -s d -l diff -d 'Compare two files with each other'
-complete -c code -s m -l merge -d 'Perform a three-way merge'
-complete -c code -s a -l add -d 'Add folder(s) to the last active window'
+complete -c code -s d -l diff -r -d 'Compare two files with each other'
```

Comment:
I'm not sure that adding `-r` here is correct. I tested

```
 code -d --wait 7z.fish 7za.fish
```

and it works for me.

I'm on 1.94.2, did it change later? If so, a link to the docs in the commit description would be nice, or at least an explanation of when things changed (though not necessarily mandatory; I'm not a maintainer here).

Reply:
Thank you for catching that.